.github/workflows/openstack/reboot_watch

```diff
@@ -10,21 +10,43 @@ use POSIX;
 my $RETVAL  = 1;
 my $RETRIES = 0;
 
+# Verify REGEX isn't already in the log before we go into tail mode.  The logodump here is sgnificantly faster than File::Tail.
+# Jump if it's found.
+
+open my $elevate_log_fh, ELEVATE_LOG_PATH;
+
+while ( !eof($elevate_log_fh) ) {
+    readline $elevate_log_fh;
+    if (/${REGEX}/) {
```

Comment:
Since we are just searching for a string of text here (the regex does not contain metacharacters, it is more secure to wrap this with \Q and \E.

```
if ( $line =~ m/\Q${REGEX}\E/ ) {
```